### DISCUSSION TITLE
Add from and select methods to Python relational API

### DISCUSSION THREAD

**Discussion by davetapley** [duckdb/duckdb#11121] [Ideas]
I quite like [Python the relational API](https://duckdb.org/docs/api/python/relational_api), but it's a bit weird that you have to start it off with e.g.:
```py
rel = con.sql('SELECT id, bar FROM foo')
```

before you can e.g.:
```py
rel.filter("id > 5")
```

Can I propose `from` and `select`, with these semantics:

`con.from('foo')` would be `select * from foo`, then:

`con.select('bar')` would change it to `select bar from foo`.

Put together:
```py
con.from('foo').select('bar').filter("id > 5")
```

**Comment by Tishj** [comment]
Replace `from` with `table` and `select` with `project` and you've got it :)

**Reply by Tishj** [reply]
Also `select` might already work, because the string gets implicitly converted into a Column expression
